tests/integration/test_token_client.py

```diff
@@ -84,6 +84,18 @@ def test_new_account(stubbed_sender, test_http_client, test_token):  # pylint: d
     assert PublicKey(account_data.owner) == stubbed_sender.public_key()
 
 
+@pytest.mark.integration
+def test_new_associated_account(test_token):  # pylint: disable=redefined-outer-name
```

Comment:
This is the test that broken. `new_acct = PublicKey(0)` won't work, you need an actual token account.

Reply:
This test works for me locally, but I also bumped up to python 3.9 as well as updated the libs -- I just pushed the Pipfile/Pipfile.lock that I am using.. perhaps this is a bug in one of the dependencies?

Reply:
The purpose of an associated token account is to derive the value for the associated token account from the main account's PublicKey using the seed and program as-shown -- it should not be generating a new private key using Pynacl when making an associated token account.

Reply:
Can we try setting `new_acct` to something else other than `PublicKey(0)`. `PublicKey(0)` is the [`SystemProgram` ](https://docs.solana.com/developing/runtime-facilities/programs#system-program) programId. I wonder if that is causing the problem. Can you try `new_acct = Account()`?

Reply:
Okay, I was able to reproduce the CI failures after removing my existing docker image with `docker image rm <IMAGE ID>`. Now why it's failing....I still don't know 😓 .